test/iron-request.html

```diff
@@ -152,6 +152,38 @@
           });
         });
 
+        test('headers are sent up', function() {
+          var options = Object.create(successfulRequestOptions);
+          options.headers = {
+            'foo': 'bar',
+            'accept': 'this should override the default'
+          };
+          request.send(options);
+          expect(server.requests.length).to.be.equal(1);
+          var fakeXhr = server.requests[0]
+          expect(fakeXhr.requestHeaders['foo']).to.be.equal(
+              'bar');
+          expect(fakeXhr.requestHeaders['accept']).to.be.equal(
+              'this should override the default');
+        });
+
+        test('headers are deduped by lowercasing', function() {
```

Comment:
Is there a corresponding test in `iron-ajax.html` that should be removed with this PR? Redundancy is fine as long as there isn't too much copy pasta ;)